.gitignore

```diff
@@ -75,6 +75,7 @@ Thumbs.db
 doc/api/generated
 doc/build
 doc/manual
+.ipynb_checkpoints
```

Comment:
Sorry to be boring - but would you mind adding the slash at the end to indicate this a directory with all contents ignored?